client/src/components/user_reviews_entry/user_reviews_entry.jsx

```diff
@@ -1,26 +1,45 @@
 import style from './user_reviews_entry.css';
+import Popup from '../flag_popup/popup.jsx';
 
 const React = require('react');
 
-const UserReviewsEntry = (props) => {
-  const dateConvert = (date) => {
+class UserReviewsEntry extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { popup: false }
+    this.dateConvert = this.dateConvert.bind(this);
+    this.clickFlag = this.clickFlag.bind(this);
+    // this.exit = this.exit.bind(this);
+  }
+
+  dateConvert(date) {
     const result = date.split(' ');
     return `${result[2]} ${result[4]}`;
-  };
-  return (
-    <div className={style.reviews}>
-      <div className={style.picture} />
-      <div className={style.name}>
-        {props.review.name}
-      </div>
-      <div className={style.date}>
-        {dateConvert(props.review.date)}
+  }
+
+  clickFlag() {
+    this.setState({ popup: !this.state.popup });
+  }
+
+  render() {
+    return (
+      <div className={style.reviews}>
+      <button className={style.flag} onClick={this.clickFlag} />
+        <div className={style.picture} />
+        <div className={style.name}>
+          {this.props.review.name}
```

Comment:
What do you think about destructuring the props to be in line with airbnb style guide?